plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AbstractJdbcBatch.java

```diff
@@ -73,6 +72,15 @@ public abstract class AbstractJdbcBatch extends Task implements JdbcStatementInt
     @PluginProperty(dynamic = true)
     private List<String> columns;
 
+    @Schema(
+        title = "The table from which columns names will be retrieved.",
+        description =
+            "This property specifies table name, which will be used to retrieve columns to specify in what columns will be inserted values. \n" +
+            "In That way columns names in insert statement will be match table schema."
+    )
```

Comment:
```suggestion
        description =
            "This property specifies the table name which will be used to retrieve the columns for the inserted values.\n" +
            "You can use it instead of specifying manually the columns in the `columns` property. In this case, the `sql` property can also be omitted, an INSERT statement would be generated automatically."
    )
```